### PR TITLE
DTGB-672: Refactored molecules and organisms component structure.

### PR DESCRIPTION
**Pull request by meirege**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have updated the documentation accordingly.
- [ ] I have updated the style guide CHANGELOG accordingly.
- [ ] I have updated the gent_base CHANGELOG accordingly.
- [ ] I have ran gulp axe on the compiled files and found no errors.

### CODE REVIEW

**Comment by delrueba** [comment]
Why is there now a h2 in a description term?
Not sure if it's permitted / advisable in terms of information overload.

**Comment by meirege** [comment]
@delrueba Seems fine to me... The H2 has been used in other components before though... I just copied it from another collated component...

File: styleguide/components/31-molecules/figure/figure.config.js [hunk 1]
```diff
@@ -4,8 +4,12 @@ module.exports = {
   name: 'Figure',
   handle: 'figure',
   status: 'alpha',
-  preview: '@preview',
   default: 'default',
+  preview: '@preview-description-list',
```
Comment (delrueba) [inline styleguide/components/31-molecules/figure/figure.config.js:8]: You need to add 'preview: '@preview'' to the variants.

File: styleguide/components/41-organisms/collection/collection.config.js [hunk 2]
```diff
@@ -5,6 +5,11 @@ module.exports = {
   status: 'alpha',
   handle: 'collection',
   default: 'grid-4',
+  preview: '@preview-description-list',
```
Comment (delrueba) [inline styleguide/components/41-organisms/collection/collection.config.js:8]: Update variants preview.

**Review by delrueba — CHANGES_REQUESTED** [review]
I would advise against using h2 tags inside a dt.
It's not correctin the document structure (there's no h1) and it's information overload.

Looks like headings are being used as styling elements.

**Comment by delrueba** [comment]
> @delrueba Seems fine to me... The H2 has been used in other components before though... I just copied it from another collated component...

I like that you're going for uniformity!
But I fear you've done it the wrong way round ;)

**Comment by meirege** [comment]
@delrueba Implemented your changes.

**Review by hstandaert — APPROVED** [review]

**Review by delrueba — APPROVED** [review]

**Pull request merged**